api/Document.json

```diff
@@ -10785,6 +10797,9 @@
               "version_added": "75"
             },
             "firefox_android": [
+              {
+                "version_added": "79"
```

Comment:
Ah, an important change! Once Samsung Internet is updated, https://developer.mozilla.org/en-US/docs/Web/API/Document/timeline#Browser_compatibility will show universal support in modern browsers.